addons/mail/static/src/core/common/discuss_migration/discuss_migration_0.js

```diff
@@ -0,0 +1,32 @@
+import { addMigration } from "./discuss_migration";
+
+export const migrationHelpers = {
+    /** @param {import("./discuss_migration").MigrationDataWithoutVersion} param */
+    add(key, shortMigrate) {
+        return addMigration({
+            key,
+            version: migrationHelpers.VERSION,
+            migrate: (...args) =>
+                Object.assign(shortMigrate(...args), {
+                    version: migrationHelpers.NEXT_VERSION,
+                }),
+        });
+    },
+    VERSION: 0,
+    NEXT_VERSION: 1,
+};
+
+migrationHelpers.add("mail.user_setting.message_sound", () => ({
+    key: "Settings,undefined<|>messageSound",
+    value: false,
+}));
```

Comment:
It seems all of them are currently trivial now, can we simplify so we don't have a callback?

Reply:
This one needs to migrate using old value: https://github.com/odoo/odoo/pull/233100/files#diff-746bdf63458963338439554f5bffafabf519e37c37489ede53eb6daf9acaaab2R13

I could support 2nd param as function or plain object.